src/ert/gui/ertwidgets/ensembleselector.py

```diff
@@ -89,6 +90,12 @@ def _ensemble_list(self) -> Iterable[Ensemble]:
             )
         else:
             ensemble_list = self.notifier.storage.ensembles
+        ensemble_list = list(ensemble_list)
```

Comment:
Interesting line to read, one would think it was already a list :sweat_smile:

Reply:
One might think so... Could keep it a generator though